src/settings/portal-settings.ts

```diff
@@ -0,0 +1,11 @@
+import { RoomVisibility } from "types/venues";
+
+type LabelOption = { label: string; value: string };
```

Comment:
No need to conserve space. A bit more expanded for visibility and diff-ability. More elaborate name due to `export`

```suggestion
export type LabelVisibilityOption = {
  label: string;
  value: string;
};
```